src/main/scala/ch/openolitor/stammdaten/repositories/StammdatenRepositoryQueries.scala

```diff
@@ -1916,7 +1916,7 @@ trait StammdatenRepositoryQueries extends LazyLogging with StammdatenDBMappings
       )
       .map((auslieferung, depots, koerbe, lieferpositionen, abos, abotypen, kunden, personen, zusatzAbos) => {
         val personenDetails = personen map (p => copyTo[Person, PersonDetail](p))
-        f(auslieferung, depots.head, koerbe, lieferpositionen, abos, abotypen, kunden, personenDetails, zusatzAbos)
+        f(auslieferung, depots.head, koerbe, lieferpositionen, abos, abotypen, kunden, personenDetails.distinct, zusatzAbos)
```

Comment:
What about abos, abotypen, kunden and zusatzAbos? I guess distinct is required for them as well. At least check if that is necessary. Thank you

Reply:
No, this is not a problem for the other lists. Actually, the problem with personen is that for every lieferposition it returns the same person with a different value for the password. This is mapped as if it was a different person. For instance: 

2 = {Person@33168} "Person(PersonId(30417),KundeId(20315),Some(Herr),Savary,Camille,Some(camillesavary@gmx.ch),None,Some(079 532 78 68),None,None,1,true,Some([C@4d6f327d),Some(2019-04-20T18:12:21.000+02:00),false,Some(KundenZugang),Set(),2016-08-06T19:40:06.000+02:00,PersonId(30028),2019-04-20T18:12:21.000+02:00,PersonId(0))"
1 = {Person@33167} "Person(PersonId(30417),KundeId(20315),Some(Herr),Savary,Camille,Some(camillesavary@gmx.ch),None,Some(079 532 78 68),None,None,1,true,Some([C@4e8a0891),Some(2019-04-20T18:12:21.000+02:00),false,Some(KundenZugang),Set(),2016-08-06T19:40:06.000+02:00,PersonId(30028),2019-04-20T18:12:21.000+02:00,PersonId(0))"

any better idea?